src/main/modules/document-download/index.ts

```diff
@@ -0,0 +1,38 @@
+import config from 'config';
+import { Application } from 'express';
+
+import { getServiceAuthToken } from '../../app/auth/service/get-service-auth-token';
+import { DocumentType } from '../../app/case/definition';
+import { AppRequest } from '../../app/controller/AppRequest';
+
+export class DocumentDownloadMiddleware {
+  public enableFor(app: Application): void {
+    const proxy = require('express-http-proxy');
```

Comment:
how comes this is set here?

Reply:
It's only used here, where would this be better set - in server.ts ?

Reply:
above like all the other packages